codespell_lib/data/dictionary.txt

```diff
@@ -6315,7 +6536,9 @@ averloaded->overloaded
 averloads->overloads
 avertising->advertising
 avgerage->average
+aviability->availability
```

Comment:
`viability` or `advisability`?

Reply:
I can't find `aviability->viability` or `aviability->advisability` or `viability->a viability`.
I found `aviable->a viable`, so I will add this.